src/uu/install/src/install.rs

```diff
@@ -564,8 +566,12 @@ fn standard(mut paths: Vec<String>, b: &Behavior) -> UResult<()> {
 
         if let Some(to_create) = to_create {
             // if the path ends in /, remove it
+            let to_create_owned;
             let to_create = if to_create.to_string_lossy().ends_with('/') {
-                Path::new(to_create.to_str().unwrap().trim_end_matches('/'))
+                let path_str = to_create.to_string_lossy();
+                let trimmed = path_str.trim_end_matches('/');
+                to_create_owned = PathBuf::from(trimmed);
+                to_create_owned.as_path()
```

Comment:
This is technically not correct, I think you need to deal with bytes.

I have some logic here: https://github.com/uutils/coreutils/blob/main/src/uu/basename/src/basename.rs#L125